tests/regression/02-base/85-evalint-maybeless.c

```diff
@@ -0,0 +1,14 @@
+// PARAM: --set ana.activated[+] expRelation
+#include <assert.h>
+#include <stddef.h>
+
+int main() {
+  int x;
+
+  // expRelation EvalInt
+  assert(!(x + 1 < x));
+  assert(!(x < x + (-1)));
+  assert(!(x - (-1) < x));
+  assert(!(x < x - 1));
```

Comment:
So `expRelation` does not consider the possibility of overflows?

Reply:
That's a good point! I think it never has, which I guess could be a problem.
Given that these cases were completely untested before and I reverse engineered the tests for them from the code, maybe we don't really need most of it. I think we'll have to separately decide what to do about it.

Reply:
Delegated to #776, as to not block this PR.